src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -55,9 +58,30 @@ export class VerifyApp extends React.Component {
     }
   }
 
+  renderVerifyButton() {
+    const { loginGovEnabled, authenticatedWithSSOe } = this.props;
+    const authVersion = authenticatedWithSSOe ? 'v1' : 'v0';
```

Comment:
v0 is deprecated and will eventually be removed.

Reply:
Also if its not much trouble can you add a space right at the `help at` so the number has a space in between.